.gitignore

```diff
@@ -78,4 +78,4 @@ tool/
 migration/sqlbindata.go
 configuration/confbindata.go
 wit/witservice
-account/tenant
+account/tenant
```

Comment:
Add `test/generated` if we are using mini mock, if not then let's forget

Reply:
are you asking me to restore the entry in `.gitignore` ?

Reply:
no, forget it as we are not generating any mock for tests.

Reply:
ok